File: cf_sdk/src/main/java/com/example/cf_sdk/logic/changebanksdk/model/member/ClientType.java
package com.example.cf_sdk.logic.changebanksdk.model.member;

import com.google.gson.annotations.SerializedName;

/**
 * Constant for device os
 */

public enum ClientType {
    @SerializedName("IOS")
    IOS("iOS"),

    @SerializedName("ANDROID")
    ANDROID("Android");

    String mClientName;

    ClientType(String clientName) {
        mClientName = clientName;
    }

    public String getClientName() {
        return mClientName;
    }
}

File: cf_sdk/src/main/java/com/example/cf_sdk/changebankapi/model/member/ClientType.kt
package com.example.cf_sdk.changebankapi.model.member

import com.google.gson.annotations.SerializedName

/**
 * Constant for device os
 */
enum class ClientType(clientName: String) {
    @SerializedName("IOS")
    IOS("iOS"), @SerializedName("ANDROID")
    ANDROID("Android");

    var clientName = ""

    init {
        this.clientName = clientName
    }
}